.github/workflows/e2e.yaml

```diff
@@ -9,14 +9,14 @@ on:
   workflow_dispatch:
 
 env:
-  HELM_VERSION: v3.13.0
-  KIND_VERSION: v0.20.0
+  HELM_VERSION: v3.15.1
+  KIND_VERSION: v0.23.0
```

Comment:
Note: we should really enable Renovate to take care of updating these dependencies.